### DISCUSSION TITLE
Loved It

### DISCUSSION THREAD

**Discussion by unignoramus11** [community/community#8448] [Copilot Conversations]
I am a student in CS and I have been using GitHub Copilot's Technical Preview for a few weeks now, and I have to say, I am seriously impressed. I have used it primarily in writing Python code for data management, and not only is it amazingly accurate in predicting the upcoming lines of my code and auto-completing the segment, but it is also greatly suggestive in making more efficient algorithms or creating new methods. It has been specifically helpful when writing code in languages that I am not that fluent in. While writing Java code, I just need to know the basic syntax and how to write comments, and it does all of the job for me. So far, I have loved it.

**Comment by SmallLemur** [comment]
Thanks for your feedback,  Could you detail a bit more how you use it in your study or personal projects? Have you encountered any issues with it ?
